apis/v1alpha1/forwarding_target_types.go

```diff
@@ -33,12 +40,37 @@ type ForwardToTarget struct {
 	// Support: Core
 	//
 	// +optional
-	TargetPort *TargetPort `json:"targetPort" protobuf:"bytes,2,opt,name=targetPort"`
+	TargetPort *TargetPort `json:"targetPort,omitempty" protobuf:"bytes,2,opt,name=targetPort"`
+
+	// Weight specifies the proportion of traffic to be forwarded to a targetRef,
+	// computed as weight/(sum of all weights in targetRefs). The following example
+	// sends 70% of traffic to service "my-trafficsplit-sv1" and 30% of the traffic
+	// to service "my-trafficsplit-sv2":
+	//
+	//   forwardTo:
+	//     - targetRef:
+	//         name: my-trafficsplit-sv1
+	//         weight: 70
+	//     - targetRef:
+	//         name: my-trafficsplit-sv2
+	//         weight: 30
+	//
+	// If only one targetRef is specified, weight is ignored and all traffic is
+	// forwarded to the targetRef. If unspecified, defaults to 1.
```

Comment:
Nit: I don't think we need this clarification. Weight doesn't need to be ignored. If there is only forwardTo it is guaranteed to include 100% of the total weight, even if that is only 1.

Reply:
@robscott commit `d7e4c20` addresses your comment above.